DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -108,16 +110,18 @@ def readSteeringFile(self):
         self.__dict__ = obj.__dict__
     self.steeringFile = os.path.abspath(sFileTemp)
 
-  def parseOptions(self):
+  def parseOptions(self, argv=sys.argv):
```

Comment:
This is bad due to the way python treats default arguments.
Use `None` as the default argument and then
```
if argv is None:
  self._argv = list(sys.argv)
```